graphql-dgs-spring-webmvc-autoconfigure/src/test/kotlin/com/netflix/graphql/dgs/webmvc/autoconfigure/DgsWebMvcConfigurationPropertiesTest.kt

```diff
@@ -16,7 +16,7 @@
 
 package com.netflix.graphql.dgs.webmvc.autoconfigure
 
-import org.assertj.core.api.Assertions
+import org.assertj.core.api.Assertions.assertThat
```

Comment:
**[UNRELATED]**: sorry for sneaking this in, but its cleaning up the `assertThat` so it does not use `Assertions.` on every `assertThat` call.

Reply:
don't mind that, I'll do it in every chance i get.